examples/pytorch/nlp/huggingface_models/text-classification/pruning/eager/README.md

```diff
@@ -1,150 +1,9 @@
-```
-
-Per-channel pruning is also supported on mrpc and sst2.
-```shell
-python3 ./run_glue_no_trainer.py \
-        --model_name_or_path "/path/to/bertmini/dense_finetuned_model" \
-        --task_name "mrpc" \
-        --max_length 128 \
-        --per_device_train_batch_size 16 \
-        --learning_rate 1e-3 \
-        --num_train_epochs 15 \
-        --weight_decay 1e-3  \
-        --do_prune \
-        --output_dir "./sparse_mrpc_bertmini" \
-        --cooldown_epochs 5 \
-        --sparsity_warm_epochs 1 \
-        --target_sparsity 0.9 \
-        --pruning_pattern "1xchannel" \
-        --pruning_frequency 50 \
-        --lr_scheduler_type "constant" \
-        --distill_loss_weight 5
-```
-```shell
-python3 ./run_glue_no_trainer.py \
-        --model_name_or_path "/path/to/bertmini/dense_finetuned_model" \
-        --task_name "sst2" \
-        --max_length 128 \
-        --per_device_train_batch_size 16 \
-        --learning_rate 5e-5 \
-        --distill_loss_weight 2.0 \
-        --num_train_epochs 15 \
-        --weight_decay 5e-5   \
-        --cooldown_epochs 5 \
-        --sparsity_warm_epochs 0 \
-        --lr_scheduler_type "constant" \
-        --do_prune \
-        --output_dir "./sparse_sst2_bertmini" \
-        --target_sparsity 0.9 \
-        --pruning_pattern "1xchannel" \
-        --pruning_frequency 500
-```
-
- Distilbert-base-uncased model pruning is supported on mrpc as following:
- ```shell
-      python run_glue_no_trainer.py \
-        --model_name_or_path "path/to/distilbert-base-uncased/dense_finetuned_model" \
-        --task_name "mrpc" \
-        --max_length 256 \
-        --per_device_train_batch_size 16 \
-        --learning_rate 1e-4\
-        --num_train_epochs 120 \
-        --weight_decay 0 \
-        --cooldown_epochs 40 \
-        --sparsity_warm_epochs 0 \
-        --lr_scheduler_type "constant" \
-        --distill_loss_weight 2 \
-        --do_prune \
-        --output_dir "./sparse_mrpc_distillbert" \
-        --target_sparsity 0.9 \
-        --pruning_pattern "4x1" \
-        --pruning_frequency 50 \
-```
-2:4 sparsity is similar to the above, only the target_sparsity and pruning_pattern need to be changed.
+MPRC and SST2 sparse models can be trained by modifying "target_sparsity" and "pruning_pattern" parameters, there are pruning scripts for different models(Bert-mini, Distilbert-base-uncased, etc) and patterns("4x1", "2:4", "1xchannel", etc).[Pruning Scripts](https://github.com/intel/neural-compressor/tree/master/examples/pytorch/nlp/huggingface_models/text-classification/pruning/eager/scripts/).
 
-To try to train a sparse model in mixed pattern, local pruning config can be set as follows:
+To try to train a sparse model in mixed pattern[Mixed-pattern Example](https://github.com/intel/neural-compressor/tree/master/examples/pytorch/nlp/huggingface_models/text-classification/pruning/eager/run_glue_no_trainer_mixed.py), local pruning config can be set as follows:
```

Comment:
mixed-patterns or mixed-pattern?

Reply:
Several pruning patterns can be mixed, so it's mixed-patterns to be exact, I will fix it.